README.md

```diff
@@ -61,6 +61,17 @@ type KbsConfigSpec struct {
   // tdxConfigSpec is the struct that hosts the TDX specific configuration
   // +optional
   TdxConfigSpec TdxConfigSpec `json:"tdxConfigSpec,omitempty"`
+
+  // ibmSEConfigSpec is the struct that hosts the IBMSE specific configuration
+  // +optional
+  IbmSEConfigSpec IbmSEConfigSpec `json:"ibmSEConfigSpec,omitempty"`
+}
+
+// IbmSEConfigSpec defines the desired state for IBMSE configuration
+type IbmSEConfigSpec struct {
+  // configPvcName is the name of the PeristentVolumeClaim where certificates/keys are mounted
+  // +optional
+  ConfigPvcName string `json:"configPvcName,omitempty"`
```

Comment:
May be the name can be changed to reflect the usage as well..  eg `CertStorePvc`